libs/cardano-ledger-api/cardano-ledger-api.cabal

```diff
@@ -0,0 +1,52 @@
+cabal-version:      2.4
+name:               cardano-ledger-api
+version:            0.1.0.0
+synopsis:           Public API for the cardano ledger codebase
+description:
+  This package a public interface into the cardano ledger codebase. It also
+  provides functionality that works for all eras and is useful for downstream
+  packages.
+bug-reports:        https://github.com/input-output-hk/cardano-ledger/issues
+
+license:            Apache-2.0
+author:             IOHK Formal Methods Team
+maintainer:         formal.methods@iohk.io
+copyright:          2021 Input Output (Hong Kong) Ltd.
```

Comment:
Do we ever update these? I see some are 2020 in the code :laughing:  But probably it doesn't matter